osu.Game.Rulesets.Catch/Edit/CatchHitObjectComposer.cs

```diff
@@ -32,6 +32,8 @@ public class CatchHitObjectComposer : HitObjectComposer<CatchHitObject>
 
         private readonly Bindable<TernaryState> distanceSnapToggle = new Bindable<TernaryState>();
 
+        protected override bool SupportsDistanceSpacing => false;
```

Comment:
I've already mentioned that I ideally wouldn't want this switch to exist, but if it has to, then maybe it should be `false` by default rather than fully abstract and having to be overridden in each consumer? Not sure. It's a ruleset API breaking change as is for no real good reason.